src/styles/shared/components/buttons/_buttons.scss

```diff
@@ -2,11 +2,20 @@
 
 @use "@hypothesis/frontend-shared/styles/mixins/focus";
 
-@use './_config' as c;
-@use './_base' as base;
+@use 'config' as c;
```

Comment:
No need to have path or underscore for local partials...it turns out. This looks cleaner.